src/App.tsx

```diff
@@ -1,7 +1,29 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string | null,
+};
+
+export class App extends React.Component {
+  state: Readonly<State> = {
+    pressedKey: null,
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', (event: KeyboardEvent) => {
+      this.setState({ pressedKey: event.key });
```

Comment:
need to write `[space]` instead of `[ ]`
<img width="199" alt="image" src="https://user-images.githubusercontent.com/55802630/182606394-70b0252d-d5dd-4d9f-b9a3-4b3016b62224.png">